debian/rules

```diff
@@ -22,6 +22,19 @@ KERNEL_TYPE.esx := generic
 KERNEL_TYPE.gcp := gcp
 KERNEL_TYPE.kvm := kvm
 
+#
+# The DELPHIX_SIGNATURE_VERSION field refers to the current Delphix
+# Release version being built and only defaults to 6.0 if not set.
+# This is passed as an environment variable via jenkins job(s) that
+# launch the linux-pkg build for this package.
+#
+DELPHIX_SIGNATURE_VERSION := $(or $(DELPHIX_SIGNATURE_VERSION),6.0)
```

Comment:
why default to 6.0? this default will soon be incorrect, after we move off the 6.0 release, so I think it might be better to just not use any default value..

Reply:
Yeah since 6.0 will be the initial release, thought it would be better to default to that and also was thinking this could avoid any flag day currently.
I'll try removing it.

Reply:
Fixed.